src/accountingservice/main.go

```diff
@@ -85,6 +87,7 @@ func main() {
 		if err := tp.Shutdown(context.Background()); err != nil {
 			log.Printf("Error shutting down tracer provider: %v", err)
```

Comment:
log.Printf -> log.Fatalf

Reply:
if we have multiple defer functions and use log.Fatalf here, it will call os.Exit(1), so I think other defer functions will not be called properly

Reply:
https://go.dev/play/p/8AHanWabshF